### ISSUE TITLE
Feature request: Generate font.css

### ISSUE THREAD

**Issue by pedrosousa13** [matthewgonzalez/fontplop#10]
Amazing tool, I do find that I need the font.css that fontprep generated with the list of all the filetypes and weights.

Like so:
``` @font-face {
  font-family: 'Neutraface 2 Text Bold';
  src: url('Neutra2Text-Bold.eot'); /* IE9 Compat Modes */
  src: url('Neutra2Text-Bold.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
       url('Neutra2Text-Bold.woff') format('woff'), /* Modern Browsers */
       url('Neutra2Text-Bold.ttf')  format('truetype'), /* Safari, Android, iOS */
       url('Neutra2Text-Bold.svg#e5bebe314e929cdf22abdf4bbfb62856') format('svg'); /* Legacy iOS */

  font-style:   normal;
  font-weight:  700;
} 
```

Would be great to have this feature :)

**Comment by wilfreddenton** [comment]
Yes, FontPrep had this and was the sole reason why I used it.

**Comment by kevinmu17** [comment]
"It is the simpler, faster, free successor to FontPrep" It's not faster if we need to code the css our selfs :)

**Comment by briangonzalez** [comment]
Fixed via #20 

cc/ @matthewgonzalez
